server/src/termdb.gdc.js

```diff
@@ -146,6 +145,8 @@ re.expand: []
 
 // prefix for keys in re._mapping{}
 const mapping_prefix = 'ssm_occurrence_centrics'
+// wait time for next check on stale case-id cache, 5min
+const cacheCheckWait = 5 * 60 * 1000
```

Comment:
@xzhou82 should make it configurable with default, `const cacheCheckWait = serverconfig.features.gdcCacheCheckWait || 5 * 60 * 1000`